2020/cfp.md

```diff
@@ -0,0 +1,57 @@
+                   Call for Participation: VMM'2020
+                   ================================
+
+                      6th Virtual Machine Meetup
+
+                         Co-located with MPLR
+                        as virtual online event
+                  November 4-6th, 2020, Manchester, UK
+                   
+                    http://vmmeetup.github.io/2020/
+
+
+The 6th Virtual Machine Meetup (VMM'2020) is a venue for discussing the latest
+research and developments in the area of managed language execution. It will be
+held on November 4-6th as a virtual event, and is colocated with the Managed Programming
+Languages & Runtimes (MPLR'20) conference (https://mplr2020.cs.manchester.ac.uk/).
+We welcome presentations of new research results, experience reports, as well as
+position statements that can lead to interesting discussions.
+
+
+Topics include, but are not limited to:
+
+ - Programming language design
+ - Dynamic and static program analysis
+ - Compiler construction
+ - Managed runtime architectures
+ - Data processing engines
+ - Distributed execution environments
+
+Participation is free of charge! To participate, please register at
+"http://vmmeetup.github.io/". There are limited participant slots due to the
+constraints of the room, so please register early, and by August 31st the latest.
+
+If you would like to give a presentation, please submit a presentation abstract 
+(a title with max. 100 characters and an abstract with max. 400 characters) to
+http://vmmeetup.github.io/.
+We may ask for additional information from you before making the program decision.
+Presentation slots are either 30 minutes (long) or 15 minutes (short)
+including Q/A.
+
+
+Important dates:
+
+ - Submissions: October 20th, 2020
+ - Author notification: October 21st, 2020
+ - Registration for participation: October 31st, 2020
+ - Virtual machine Meetup: November 4-6th online (via Zoom)
+
+Submission link:
+
+ - http://vmmeetup.github.io/
+
+Program committee:
+
+ - David Leopoldseder, Johannes Kepler University, Linz
```

Comment:
labs I am ;-)

Reply:
I liked the idea we had reviewers from different institutions ;-) but fixed

Reply:
but that would be a lie :D